adapt to new key-contacts (#3793)
deltachat/deltachat-android: CHANGELOG.md

* Improve hint for app drafts
* Add Text-To-Speech (TTS) support for in-chat apps
* New icon for the QR icon
* Improved separation between unencryted chats/contacts and encrypted ones, avoiding mixing of encrypted and unencrypted messages in the same chat

> * Removed padlocks, as encrypted is the default "normal" state, instead, unencryted email is marked as such
> * Classic email chats/threads now have an email letter icon making it easy to recognize

Reviewer: Some suggestions:
- Start a new sentence after `"normal" state`
- Typo: unencryted -> unencrypted
- Explain how unencrypted email is marked

```suggestion
* Removed padlocks, as encrypted is the default "normal" state. Instead, unencrypted email is marked with a small email / letter  (✉️) icon
* Classic email chats/threads get a big email / letter icon making it easy to recognize
```
